new-components/reporters/defectdojo/cmd/main.go

```diff
@@ -0,0 +1,43 @@
+package main
+
+import (
+	"context"
+	"log"
+	"time"
+
+	"github.com/go-errors/errors"
+
+	"github.com/smithy-security/smithy/sdk/component"
+
+	"github.com/smithy-security/smithy/new-components/reporters/defectdojo/internal/client"
+	"github.com/smithy-security/smithy/new-components/reporters/defectdojo/internal/reporter"
+)
+
+func main() {
+	ctx, cancel := context.WithTimeout(context.Background(), 1*time.Minute)
+	defer cancel()
+
+	if err := Main(ctx); err != nil {
+		log.Fatalf("unexpected error: %v", err)
+	}
+}
+
+func Main(ctx context.Context, opts ...component.RunnerOption) error {
+	opts = append(opts, component.RunnerWithComponentName("Defectdojo"))
```

Comment:
lets keep this lowecase with _ plz

Reply:
it's one word so just lowercasing will do